src/includes/getting-started-install/android.mdx

```diff
@@ -1,4 +1,34 @@
-To install the Android SDK, add it your `build.gradle` file:
+### Auto-Installation With the Sentry Android Gradle Plugin
+
+To install the plugin, add it to your `build.gradle` file:
+
+```groovy
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+plugins {
+  id "io.sentry.android.gradle" version "3.1.0-alpha.1"
+}
+```
+
+```kotlin
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+plugins {
+  id("io.sentry.android.gradle") version "3.1.0-alpha.1"
+}
+```
+
+The plugin will automatically add the latest version of the Android SDK as a dependency.
+
+### Manual Installation
+
+To install the Android SDK, add it to your `build.gradle` file:
```

Comment:
Suggestion for rewording so it's more clear that this is an option, other than the Gradle one:

```suggestion
If you don't want the Sentry Gradle plugin to install the Sentry SDK automatically for you, you can define the dependency directly to your `build.gradle` file:
```